test/controllers/offer.controller.test.js

```diff
@@ -52,7 +65,8 @@ describe('Offer Controller', () => {
             expect(res.body.offer).to.have.property('userInfo');
             expect(res.body.offer.vendorInfo._id).to.be.eql(adminId.toString());
             expect(res.body.offer.vendorInfo._id).to.be.eql(adminId.toString());
-            expect(res.body.offer.propertyInfo._id).to.be.eql(propertyId.toString());
+            expect(res.body.offer.propertyInfo._id).to.be.eql(createPropertyId.toString());
+            expect(res.body.offer.referenceCode.substr(0, 9)).to.be.eql('LVE/OLM/1');
```

Comment:
From the mail sent by Nnamdi, the number sold should be a minimum of 2 digits. Here is an excerpt from the mail. Simply write a function in the helper file to fix this.

```
If its the third maisonette sold that day - 03
```
```suggestion
            expect(res.body.offer.referenceCode.substr(0, 9)).to.be.eql('LVE/OLM/01XXXXXX');
```